### ISSUE TITLE
[MAS4.1.2][Screen reader-choose a different location.] Role is incorrectly defined for Browse controls 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1778]
**Actual Result:**
Role is incorrectly defined for Browse controls present in the Resource settings for this bot dialog 
Role is defined as link instead of button 

**Expected Result:**
Role should be defined as button for browse controls and narrator should narrate as Browse button

**Repro Steps:**
Turn on the narrator
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the resource’s icon left side of the pane and select it 
3.	Navigate to the Transcripts button and select it 
4.	Navigate to the choose a different location and select it 
5.	Dialog gest opened 
6.	Navigate to the browse controls 
7.	Observe the issue
 
**User Impact:**
Users who rely on screen reader will be affected to navigate the control if role is incorrectly defined for buttons.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
![MAS4 1 2_Choose file location](https://user-images.githubusercontent.com/53518994/63590677-2e28c400-c5ca-11e9-8d31-2e35b5253e30.png)

**Comment by tonyanziano** [comment]
This will be fixed by #1828

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 with NVDA(2019.2) .Issue still reproing hence reactivating this bug .
NVDA is reading as Browse link only .

**Comment by corinagum** [comment]
Remaining fix should be resolved by #1877. Will re-confirm when merge is in.

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version 4.5.2-80829 with NVDA(2019.2) issue got fixed now hence we are closing this bug
